images/diagnostics/scripts/openshift-network-debug

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# This script is used by network diagnostics.
+# Based on containerized/non-containerized openshift install,
+# it sets the environment so that docker, openshift, iptables, etc.
+# binaries are availble for network diagnostics.
+#
+set -o nounset
+set -o pipefail
+
+node_rootfs=$1
+shift
+cmd="${@}"
+node_image="openshift/node"
+
+node_container_id=$(chroot "${node_rootfs}" docker ps | grep "${node_image}:" | cut -d' ' -f1)
+
+if [[ -z "${node_container_id}" ]]; then # non-containerized openshift env
+
+    chroot "${node_rootfs}" ${cmd}
+
+else # containerized env
+
+    # On containerized install, docker on the host is used by node container,
+    # For the privileged network diagnostics pod to use all the binaries on the node:
+    # - Copy kubeconfig secret to node mount namespace
+    # - Run openshift under the mount namespace of node
+    node_docker_pid=$(chroot "${node_rootfs}" docker inspect "${node_container_id}" | grep Pid | head -n 1 | cut -d':' -f2 | cut -d',' -f1)
+    data=$(cat "${node_rootfs}"/secrets/kubeconfig)
+
+    chroot "${node_rootfs}"  nsenter -m -t "${node_docker_pid}" echo "${data}" > /kubeconfig
+    chroot "${node_rootfs}" nsenter -m -t "${node_docker_pid}" sh -c 'KUBECONFIG=/kubeconfig '"$cmd"''
```

Comment:
the first line has an extra space which makes the lines not line up. also, add `--` between the nsenter args and the command, for clarity?

I think you can pass "-r" to nsenter and then not need to use both chroot and nsenter?

The "> /kubeconfig" is evaluated outside of the chroot, so it's referring to the real rootfs not the node_rootfs

also... cmd ends up getting parsed again here